CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 * Update maven archetype for project generation with new client recommendations.
 * Update client documentation to recommend using an abstracted resource pattern.
 
+### Defects Corrected
+* Exclude JAX-RS providers with binding annotations to be registered in the resteasy provider factory for the server.
```

Comment:
With the 3.3 release we are switching to the Keep a Changelog style for the Changelog - https://keepachangelog.com/en/1.0.0/

This section would become `Fixed`. The sentence structure may change a tiny bit too, there are some examples on the link above.

Reply:
https://github.com/cerner/beadledom/pull/143/commits/75d364f068be5b48f12a864aa2a8bd938fb929b8